account_multicurrency_revaluation_type/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Multicurrency revaluation",
+    "version": "12.0.1.0.1",
+    "category": "Finance",
+    "summary": "Manage revaluation for multicurrency environment",
```

Comment:
IMO "Use monthly currency rate for multicurrency revaluation" would be closer to what the module does.